modules/http/src/main/java/org/mule/module/http/internal/listener/HttpRequestToMuleEvent.java

```diff
@@ -122,6 +122,15 @@ else if (entity instanceof InputStreamHttpEntity)
         }
 
         final DefaultMuleMessage defaultMuleMessage = new DefaultMuleMessage(payload, inboundProperties, outboundProperties, inboundAttachments, muleContext);
+
+        for (String property : inboundProperties.keySet())
```

Comment:
Why not use "request.getHeaderValue(HttpHeaders.Names.CONTENT_TYPE);" as used already on line 86.  Of move line 86 up so it's in scope?